kubernetes/kubernetes/src/test/java/org/arquillian/cube/kubernetes/ResourceGeneratorBuilderIT.java

```diff
@@ -0,0 +1,40 @@
+package org.arquillian.cube.kubernetes;
+
+import java.io.File;
+import java.nio.file.Paths;
+import org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift;
+import org.arquillian.smart.testing.rules.git.GitClone;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Category(RequiresOpenshift.class)
+@RequiresOpenshift
+public class ResourceGeneratorBuilderIT {
+
+    @ClassRule
+    public static final GitClone gitClone = new GitClone("https://github.com/snowdrop/spring-boot-http-booster");
```

Comment:
Can't we make it local as a bundle? We know that at this point of time this repo has all we need, but it might not be the case in the future. So if we really want to use git for this test we should lock it down to a known state.

But then it opens a question - do we really need to use git for this tests? Can't we have some super tiny project as test-resource folder instead? Did you consider such approach?

Reply:
Good idea.Tiny project as test-resource folder would work for this.